lib/routes/charge/cart/cart_bar_drop_down_button.dart

```diff
@@ -20,50 +21,77 @@ class CartBarDropDownButton extends StatelessWidget {
     Key key,
   }) : super(key: key);
 
+  @override
+  State<CartBarDropDownButton> createState() => _CartBarDropDownButtonState();
+}
+
+class _CartBarDropDownButtonState extends State<CartBarDropDownButton> {
+  List<CurrencyWrapper> currencyList = [];
+
+  @override
+  void initState() {
+    super.initState();
+    WidgetsBinding.instance.addPostFrameCallback((_) {
```

Comment:
Do we need the postFrameCallback?

Reply:
No, it's residue code from another implementation. Removing.